ovos_persona/__init__.py

```
@@ -105,17 +110,26 @@
+            self.intent_matchers[lang] = IntentContainer(cache_dir=f"{intent_cache}/{lang}") \
+                if IS_PADATIOUS else IntentContainer()
             for intent_name in ["ask.intent", "summon.intent"]:
                 samples = intent_data.get(intent_name)
                 samples = flatten_list([expand_template(s) for s in samples])
                 if samples:
                     LOG.debug(f"registering Persona intent: {intent_name}")
-                    self.intent_matchers[lang].add_intent(
-                        intent_name.replace(".intent", ""), samples)
+                    try:
+                        self.intent_matchers[lang].add_intent(intent_name, samples)
+                    except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)